pages/project-list-empty-state.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Component Kit</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="../style.css">
+	</head>
+	<body>
+		<a class="u-sr-only u-skip-link" href="#content">Skip to content</a>
+		<header class="c-site-header">
+			<nav class="c-site-header__nav">
+				<img src="../images/logos/reflectivity-icon.svg" alt="Reflectivity logo" width="22" height="32" class="c-site-header__logo">
+				<ul class="o-list-bare c-tabs">
+					<li class="c-tabs__item">
+						<a class="c-tabs__link c-tabs__link--ko is-selected" href="#">Projects</a>
+					</li>
+				</ul>
+			</nav>
+			<nav class="c-site-header__nav">
+				<ul class="o-list-bare c-tabs">
+					
+					<li class="c-tabs__item">
+						<button class="o-button-bare c-button--dropdown c-tabs__link c-tabs__link--ko">
+							<span class="u-text-truncate">CC</span>
+						</button>
+						<div class="o-container o-container--xxs c-dropdown" data-position="bottom-end">
+							<div class="o-box o-box--small">
+								<p class="u-text-truncate">
+									Signed in as<br>
+									<strong>Carolyn Chapman</strong>
+								</p>
+							</div>
+							<ul class="o-list-bare">
+								<li><a href="login.html" class="c-list-link c-list-link--small">Sign Out</a></li>
+							</ul>
+						</div>
+					</li>
+				</ul>
+			</nav>
+		</header>
+		<header class="c-page-header">
+			<div class="o-box">
+				<h1>Projects</h1>
+			</div>
+		</header>
+		<main id="content">
+			<div class="u-flex u-align-items-center u-justify-content-center u-margin-top-lg">
```

Comment:
You can use one of the [`o-box` classes](https://github.com/philbuchanan/component-kit/blob/master/scss/objects/_objects.box.scss#L30) instead of a utility class to add a bunch of padding here. If you need an `xl` variant, you can add that too.

Might also be a good idea to change all the box classes to `sm`, `md`, `lg` to be consistent with other suffixes. I'll make a ticket for that.

Reply:
Updated